runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -389,6 +414,51 @@ public void onChatMessage(ChatMessage event)
 		}
 	}
 
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
+	{
+		if (event.getItemContainer() != client.getItemContainer(InventoryID.INVENTORY))
+		{
+			return;
+		}
+		
+		if (eventType != null && eventType.equals(BRIMSTONE_CHEST_EVENT_TYPE))
+		{
+			onBrimstoneChestLooted(event.getItemContainer());
+			eventType = null;
+		}
+	}
+
+	private void onBrimstoneChestLooted(ItemContainer inventoryContainer)
+	{
+		if (inventoryContainer != null && inventorySnapshot != null)
+		{
+			Multiset<Integer> currentInventory = HashMultiset.create();
+			Arrays.stream(inventoryContainer.getItems())
+					.forEach(item -> currentInventory.add(item.getId(), item.getQuantity()));
+
+			final Multiset<Integer> diff = Multisets.difference(currentInventory, inventorySnapshot);
+
+			diff.forEachEntry(this::submitBrimstoneChestLoot);
```

Comment:
This submits one loot entry per item... is there always just one item?

Reply:
according to the wiki everything dropped is either noted, stackable, or dropped in a quantity of 1. So yes